validation-test/stdlib/RuntimeSymbols.swift

```diff
@@ -1,6 +1,6 @@
 # Verify symbols from libswiftCore.dylib.
 
-UNSUPPORTED: OS=linux
+UNSUPPORTED: OS=linux-gnu
```

Comment:
For now we won't. I don't think there is a common set of parameters to `nm` that do what we want (`-U` vs `--defined-only`), so we would need a second test with a Linux-compatible `nm` command.